beanstalk_worker/worker_start_build.py

```diff
@@ -31,88 +31,95 @@
 DEBUG=1
 
 def debug_log(msg):
-  if DEBUG==1:
-    logger.log(level=logging.INFO,msg=msg)
+    if DEBUG==1:
+        logger.log(level=logging.INFO,msg=msg)
 
 def run_command(command):
-  p = Popen(command,bufsize=0,shell=True,stdout=PIPE,stderr=PIPE,stdin=PIPE)
-  p.wait()
-  out = p.communicate()
-  return out
+    p = Popen(command,bufsize=0,shell=True,stdout=PIPE,stderr=PIPE,stdin=PIPE)
```

Comment:
thanks @navidshaikh I would change the whole function in next clean up PR. do not want to add more changes to this PR.